src/devtools/client/debugger/src/components/shared/Popover.tsx

```diff
@@ -1,310 +1,138 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
-
-//
-import React, { Component } from "react";
-import ReactDOM from "react-dom";
-
-import classNames from "classnames";
-import BracketArrow from "./BracketArrow";
-import type { BracketArrowOrientation, BAProps } from "./BracketArrow";
-import SmartGap from "./SmartGap";
-
-interface PopoverProps {
-  type: "tooltip" | "popover";
```

Comment:
The old "tooltip" type wan't being used, but it's kind of akin to `showTail={false}` in the new component so far as I **know.**